conjure-java-core/src/main/java/com/palantir/conjure/java/services/dialogue/DialogueServiceGenerator.java

```diff
@@ -49,10 +49,14 @@ public DialogueServiceGenerator(Options options, String apiVersion) {
 
         TypeMapper parameterTypes = new TypeMapper(
                 conjureDefinition.getTypes(),
-                new ClassVisitor(conjureDefinition.getTypes(), options, ClassVisitor.Mode.PARAMETER));
+                new DialogueClassNameVisitor(
```

Comment:
its a bit confusing to have a `DialogueClassNameVisitor` and a `ClassVisitor` both of which encode dialogue specific details.

Reply:
Ya, I didn't grok that ClassVisitor was dialogue specific, I can combine the two. Mind if I rename it to include dialogue?

Reply:
please do!

Reply:
Should be good to go